### DISCUSSION TITLE
Listing Logstash Plugins

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#1658] [Unsupported Versions] [answered]
Trying to find which logstash plugins are available by default in SO. Logstash documentation mentions "bin/logstash-plugin list" as the way to do this. Is this tool available in SO 2.3.1?

**Comment by dougburks** [comment, marked as answer]
You can run the command as follows:
```
sudo docker exec -it so-logstash bin/logstash-plugin list
```

However, please note that we've moved all parsing from Logstash to Elasticsearch:
https://docs.securityonion.net/en/2.3/logstash.html#adding-new-logs-or-modifying-existing-parsing
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing
